### DISCUSSION TITLE
Question: What is the default libpath directory on Mac?

### DISCUSSION THREAD

**Discussion by isakube** [HaxeFlixel/flixel#2312] [Q&A] [answered]
I may or may not have accidentally relocated my libpath directory, and I can't find where it used to be. Does anybody know?

**Comment by RichardBray** [comment, marked as answer]
I think it's just `/Users/<your_mac_username>/haxe/haxelib` If you've got a haxelib package. There's also `haxe_libraries` in that same haxe directory which I think is just for non haxelib packages 🤷‍♂️ but don't quote me on that.

**Reply by isakube** [reply]
aight thanks

**Comment by Geokureli** [comment]
Mine has always been `/usr/local/lib/haxe/lib`
